src/util/time.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Redis Ltd. 2018 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+
+#include "time.h"
+#include <stddef.h>
+#include <sys/time.h>
+
+uint64_t get_unix_timestamp_milliseconds(void) {
```

Comment:
Please include comments from header
```c
// Returns the milliseconds elapsed since the UNIX epoch.
 uint64_t get_unix_timestamp_milliseconds(void) {
```

Reply:
Thank you for the detailed comment, that's simply a convention I'm trying to follow,
I find it handy to have the docs/comments in both the `.c` and `.h` files, the text edit I'm using doesn't have this functionlity (at least not out of the box)